src/pyscript/plugins/create.py

```diff
@@ -20,25 +66,76 @@ def create(
         "--project-type",
         help="Type of project that is being created. Supported types are: 'app' or 'plugin'",
     ),
+    wrap: Optional[bool] = typer.Option(
+        False,
+        "-w",
+        "--wrap",
+        help="Use wrap mode i.e. embed a python script into an HTML file",
+    ),
+    output: Optional[Path] = typer.Option(
+        None,
+        "-o",
+        "--output",
+        help="""Path to the resulting HTML output file. Defaults to input_file with suffix replaced.
+        Meant to be used with `--wrap`""",
+    ),
+    command: Optional[str] = typer.Option(
+        None,
+        "-c",
+        "--command",
+        help="If provided, embed a single command string. Meant to be used with `--wrap`",
+    ),
+    title: Optional[str] = typer.Option(
+        None, help="Add title to HTML file. Meant to be used with `--wrap`"
+    ),
+    show: Optional[bool] = typer.Option(
+        False,
+        "--show",
+        help="Open output file in web browser. Meant to be used with `--wrap`",
+    ),
 ):
     """
     Create a new pyscript project with the passed in name, creating a new
-    directory in the current directory.
+    directory in the current directory. Alternatively, use `--wrap` so as to embed
+    a python file instead.
     """
-    try:
-        create_project(
-            app_name,
-            app_description,
-            author_name,
-            author_email,
-            pyscript_version,
-            project_type,
+    if not app_or_file_name and not command:
+        raise cli.Abort(
+            "Must provide either an input '.py' file or a command with the '-c' option."
         )
-    except FileExistsError:
+
+    if app_or_file_name and command:
+        raise cli.Abort("Cannot provide both an input '.py' file and '-c' option.")
+
+    if (title or output or show or command) and (not wrap):
         raise cli.Abort(
-            f"A directory called {app_name} already exists in this location."
+            """`--title`, `--output/-o`, `--show` and `--command/-c`
+            are meant to be used with `--wrap/-w`"""
         )
 
+    if not wrap:
+        if not app_description:
+            app_description = typer.prompt("App description")
+        if not author_name:
+            author_name = typer.prompt("Author name")
+        if not author_email:
+            author_email = typer.prompt("Author email")
+        try:
```

Comment:
I do think we need to revisit these options above in a different PR/Discussion but, assuming we keep things as is, why we don't use them for wrap?

Reply:
because the idea of the original `pyscript wrap` was just to spit out an HTML file. And not really a project folder.

I haven't changed the functionality, merely borrowed it and subsumed it under `create`

Reply:
We are now using them for `wrap` as well in the new implementation :D